src/index.js

```diff
@@ -1,34 +1,151 @@
+    } else {
+      localData[i].completed = false;
+    }
+  }
+  localStorage.setItem('list', JSON.stringify(localData));
+};
+const myArray = [];
+const addTodo = (todoValue) => {
+  const todoContainer = document.createElement('div');
+  todoContainer.className = 'todoContainer';
+  todoContainer.innerHTML += `
+      <input type="checkbox" class="checkbox">
+      <span>${todoValue}</span>
+      <i class="fas fa-ellipsis-v"></i>
+      <i class="fas fa-trash-alt"></i>
+    `;
+  todosMaincontainer.appendChild(todoContainer);
+
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((i) => {
+    i.addEventListener('click', () => {
+      i.parentElement.classList.toggle('checkedContainer');
+      i.nextElementSibling.classList.toggle('checkTodo');
+      i.parentElement.lastElementChild.classList.toggle('trash-active');
+      i.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-desable');
+      updateLocal();
+    });
+  });
+  /* eslint-disable */
+  const editTodo = (todoContainer, todo) => {
+    const editInput = document.createElement('input');
+    editInput.type = 'text';
+    editInput.className = 'editInput';
+    editInput.value = todo.textContent;
+    todoContainer.replaceChild(editInput, todo);
+    editInput.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter') {
+        const todoContainers = document.querySelectorAll('.todoContainer');
+        const localData = JSON.parse(localStorage.getItem('list'));
+        for (let i = 0; i < todoContainers.length; i += 1) {
+          if (todoContainers[i].classList.contains('checkedContainer')) {
+            localData[i].description = editInput.value;
+            localStorage.setItem('list', JSON.stringify(localData));
+          }
+        }
+  
+        editInput.parentElement.classList.remove('checkedContainer');
+        todoContainer.replaceChild(todo, editInput);
+        todo.textContent = editInput.value;
+      }
+    });
+  };
+  const object = new MyObject(todoValue, false, checkbox.length - 1);
+  myArray.push(object);
+  localStorage.setItem('list', JSON.stringify(myArray));
+
+  const editIcons = document.querySelectorAll('.fa-ellipsis-v');
+  editIcons.forEach((i) => {
+    i.addEventListener('click', () => {
+      i.parentElement.classList.add('checkedContainer');
+      editTodo(todoContainer, i.previousElementSibling);
+    });
+  });
+
+  const removeIcons = document.querySelectorAll('.fa-trash-alt');
+  removeIcons.forEach((i) => {
+    i.addEventListener('click', () => {
+      removeTodo(i.parentElement);
+    });
+  });
+};
+ /* eslint-disable */
+const removeTodo = (todo) => {
+  todosMaincontainer.removeChild(todo);
+  let count = 0;
+  const localData = JSON.parse(localStorage.getItem('list'));
+  const data = Array.from(localData).filter((i) => i.completed === false);
+  data.map((i) => i.index = count += 1);
+  localStorage.setItem('list', JSON.stringify(data));
+};
```

Comment:
- [ ] please make sure  the task is removable at the moment when I click on the trash the task is not removed pleas try to fix that